antelope_contracts/contracts/evmutil/src/evmutil.cpp

```diff
@@ -141,7 +141,7 @@ void evmutil::dpystakeimpl() {
     require_auth(get_self());
 
 
-    uint64_t id = 0;
+    uint64_t id = 9;
```

Comment:
We should delete this check instead of change the id to 9

Reply:
OK，got it.

Reply:
Has done.